docs/guides/api.en.md

```diff
@@ -345,7 +471,344 @@
 
 ---
 
-## Patching : Stage 1 (Loading)
+## Settings
+
+??? example "AddMenu `method`"
+    <h3>Summary</h3>
+    Adds an entry to the in-game mod menu, which lets users configure settings at runtime.
+    <h3>Example</h3>
+    ```c#
+    // Example : Adds a slider to the mod menu to configure the amount of tiles walked by turn.
+    // The slider will have a range from 2 to 5, with a default value of 4.
```

Comment:
Not sure if the example is clear or not. It does not configure the amount of tiles walked by turn but change the value holds by global.tiles_by_turn and the text displaid is "Tiles walked by turn".

Maybe using a non specific example would be better

Reply:
More of an inaccuracy on what it does, but I agree, it would be better.